src/test/java/net/sf/javaanpr/recognizer/CharacterRecognizerTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2013 JavaANPR contributors
+ * Copyright 2006 Ondrej Martinsky
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.osedu.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package net.sf.javaanpr.recognizer;
+
+import org.junit.Test;
+
+import java.util.Vector;
+
+import static org.junit.Assert.*;
+import static org.hamcrest.CoreMatchers.*;
```

Comment:
I think it would be better if we used raw junit for assertions, if it's not too much of a pain. As far as I can tell, all the assertThat(..) statements in this test can be substituted with assertEquals.
If in the future we find out that we need Hamcrest's matchers, lets use them and add them to our dependencies.